app/javascript/chat/chat.jsx

```diff
@@ -1409,7 +1410,44 @@ export default class Chat extends Component {
       .getElementById('jumpback_button')
       .classList.remove('chatchanneljumpback__hide');
   };
+  handleImageDrop = (e) => {
+    e.preventDefault();
+    const { files } = e.dataTransfer;
 
+    const messageArea = document.getElementById('messagelist');
```

Comment:
This can be improved by using a `ref` as mentioned previously, but not a blocker.